### PR TITLE
Change the base package to com.gooddata.sdk.common

### LINKED ISSUE #52: Change base package to something unique
We should update base package to something like `com.gooddata.sdk.common` to be compatible with jigsaw.

### PR DESCRIPTION
**Pull request by jimirocks**
Resolves #52

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: src/test/groovy/com/gooddata/sdk/common/UriPrefixerTest.groovy [hunk 1]
```diff
@@ -3,8 +3,9 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE.txt file in the root directory of this source tree.
  */
-package com.gooddata
+package com.gooddata.sdk.common
 
+import com.gooddata.sdk.common.UriPrefixer
```
Comment (peter-plochan) [inline src/test/groovy/com/gooddata/sdk/common/UriPrefixerTest.groovy]: Remove unnecessary import

File: src/test/groovy/com/gooddata/sdk/common/UriPrefixingClientHttpRequestFactoryTest.groovy [hunk 2]
```diff
@@ -3,8 +3,9 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE.txt file in the root directory of this source tree.
  */
-package com.gooddata
+package com.gooddata.sdk.common
 
+import com.gooddata.sdk.common.UriPrefixingClientHttpRequestFactory
```
Comment (peter-plochan) [inline src/test/groovy/com/gooddata/sdk/common/UriPrefixingClientHttpRequestFactoryTest.groovy]: Remove unnecessary import

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-rest-common-merge-compliance-check/25/ : SUCCESS in 16s

**Pull request merged**